geo/src/algorithm/convex_hull/qhull.rs

```diff
@@ -42,9 +42,8 @@ where
         // (2) max_idx = min_idx: then any point could be
         // chosen as max. But from case (1), it could now be
         // 0, and we should not decrement it.
-        if max_idx > 0 {
-            max_idx -= 1;
-        }
+        max_idx = max_idx.saturating_sub(1);
```

Comment:
Unclear if this is an improvement... 😒📎

Reply:
I like it better, if only because you have immediate access to what saturating_sub does if you're using an IDE / LSP

Reply:
But.. you don't need to read do docs to understand -= 1 either 😀.

Reply:
This entire commit is a bikeshed honeypot / nerd snipe. Everybody be on your best behavior.